docs/explanations/modeling_extensions/surrogate/alamopy/alamopy-depr.rst

```diff
@@ -0,0 +1,167 @@
+.. alamopy documentation master file, created by
```

Comment:
Drop all of this - we do not want to advertise the old interface, as it is only there to retain internal functionality for RIPE and HELMET.